packages/pn-pa-webapp/src/components/Statistics/__test__/FilterStatistics.test.tsx

```diff
@@ -22,17 +21,20 @@ vi.mock('react-i18next', () => ({
   }),
 }));
 
+const lastDate = new Date('2024-03-14T00:00:00.000Z');
+
 const last6MonthsFilterValue: StatisticsFilter = {
   startDate: sixMonthsAgo,
-  endDate: today,
+  endDate: lastDate,
```

Comment:
Why we replace today with this? I would expect that these test works without this change. Try to restore the old version and add a test case with lastDate where you check that the end date is populated with this value